### ISSUE TITLE
Why does toISO return date in UTC?

### ISSUE THREAD

**Issue by lbjones** [moment/luxon#257]
I seem to be going crazy, this can't be a bug, I must be doing something wrong.

`console.log(DateTime.local().zone.name); // prints 'America/Denver'`

However when I do this:

`console.log(DateTime.local().toISO()); // prints '2018-05-12T14:17:55.203+00:00'`

Why is it showing the time in UTC??

The example in the docs show the following:

`DateTime.local().toISO() //=> '2017-04-22T20:47:05.335-04:00'`

I could use some form of .toLocaleString() but that seems totally wrong.

**Comment by icambron** [comment]
I expect that to work as you describe, and in the general case, it definitely does, like this:

```js
> DateTime.local().toISO()
'2018-05-13T15:59:41.611-04:00'
```

Where is this running? (e.g. browser and version). And what version of Luxon?

**Comment by lbjones** [comment]
I'm using the latest version of Chrome on Luxon 1.2.0 on my Mac laptop.

I just tried it on my desktop and it worked as expected.  So I went back to the laptop and tried to discover what the difference was.  I restarted the browser and it worked as expected!  

```
> DateTime.local().toISO()
'2018-05-14T07:23:21.614-06:00'
```

I don't think Chrome had updated since the last time I started it up, which was likely a week or so ago.  So I'm afraid I can't provide any useful info.  I'll just have to chalk it up to a browser bug.  

Thanks for your help.  This library is awesome by the way!
